daemon/info.go

```diff
@@ -133,9 +133,13 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		v.CPUSet = sysInfo.Cpuset
 	}
 
-	if hostname, err := os.Hostname(); err == nil {
-		v.Name = hostname
+	hostname := "<unknown>"
```

Comment:
@thaJeztah 
I tried on my ubuntu server that when hostname is `<unknown>`, executing command `hostname`,the system will show `(none)`.
Seems `<unknown>` is an invalid value.